storybook/stories/API/Accessibility.mdx

```diff
@@ -21,9 +21,12 @@ To see how this works, try the following chart, where the accessibility layer ha
         <XAxis dataKey="name" />
         <YAxis />
         <Tooltip />
+        <Brush />
     </LineChart>
 </ResponsiveContainer>
 
+This chart also contains a "brush", a range slider that lets you control what appears on the X-axis. You can tab to the "travellers" on the brush, and use the left/right arrow keys to adjust them.
```

Comment:
nit: the part below "the code to generate this example is"... Is missing the `Brush`